### PR TITLE
Optimize validator docs

### PR DESCRIPTION
**Pull request by memorycancel**
As the title: Optimize validator docs.

### CODE REVIEW

File: src/cn/maintain/validator-stake.md [hunk 1]
```diff
@@ -37,4 +37,8 @@
 
 解绑期结束后，您可以通过点击`My Staking`面板中的`Withdraw Stakes`按钮来取回质押。
 
-![withdraw stakes](../../maintain/withdraw_stakes.jpg)
\ No newline at end of file
+![withdraw stakes](../../maintain/withdraw_stakes.jpg)
+
+如果您打算停止验证节点，首先您**必须**执行 Unbond 操作并保持验证器节点运行，直到下一轮验证器上线（下一个era结束）。
```
Comment (lesterli) [inline src/cn/maintain/validator-stake.md]: 验证器节点 -> 验证节点
下一轮验证器 -> 下一轮验证人
下一个era结束 -> 当前 Era 结束

File: src/cn/maintain/validator-stake.md [hunk 2]
```diff
@@ -37,4 +37,8 @@
...
+
+对于一键部署的节点，您可以在 **My Node** 面板中进行 **Destory** 操作。对于手动部署，您可以手动停止验证器节点。
```
Comment (lesterli) [inline src/cn/maintain/validator-stake.md]: 验证器节点 -> 验证节点

File: src/maintain/validator-deploy-manually.md [hunk 3]
```diff
@@ -80,4 +80,8 @@ You can check whether the validator node complete the synchronization of the cha
 2021-09-21 00:13:18 ✨ Imported #54434 (0xba36…ee68)
 2021-09-21 00:13:18 [54434] 🐙 Current block: 54434 (parent hash: 0x84aa3d1b6455859f9503d6ecc70b50b183141fe08f5b0695357e00fe1d24d915)
 2021-09-21 00:13:18 💤 Idle (6 peers), best: #54434 (0xba36…ee68), finalized #54431 (0xd194…b319), ⬇ 22.0kiB/s ⬆ 21.9kiB/s
-```
\ No newline at end of file
+```
+
+Previous：[Generate Validator Account](./maintain/validator-generate-keys.html)
+
```
Comment (lesterli) [inline src/maintain/validator-deploy-manually.md:86]: Previous -> Previous step
Next -> Next step

File: src/maintain/validator-stake.md [hunk 4]
```diff
@@ -44,6 +44,6 @@ After the unbonding period, you can withdraw your stakes via clicking the `Withd
 
 ![withdraw stakes](../maintain/withdraw_stakes.jpg)
 
-If you plan to stop the validator node, firstly you **MUST** do the `Unbond` operation and keep the validator node running until the next round validators comes online.
+If you plan to stop the validator node, firstly you **MUST** do the `Unbond` operation and keep the validator node running until the next round validators comes online（next era ended）.
```
Comment (lesterli) [inline src/maintain/validator-stake.md]: next era ended -> current era ended

**Comment by memorycancel** [comment]
@lesterli Changed as your suggestions, thanks, please review again.

**Pull request merged**